client/src/common/components/Dialog.scss

```diff
@@ -37,10 +37,16 @@
 
   &__children {
     position: relative;
-    font-size: 13px;
+    font-size: $s-font;
     line-height: 1.2;
     color: $text-normal;
     padding: four-by(4) four-by(4) 0;
+
+    span {
```

Comment:
Please remove this as it can lead to many unexpected side-effects and investigate white-space prop.

Reply:
I added `white-space: pre`, `white-space: pre-line`, `white-space: pre-wrap` to paragraph to `&__children` wrapper an it didn't work. This was because I didn't add any space between those spans: 
```
 <p>
     <FormattedMessage
       id={externalAction.messageId}
       values={{ name, performer: externalAction.performer }}
    />
       {!isOwner && (
           <Fragment>
             {' '}
              <FormattedMessage
                  id="cohort.actions.not-available-contact-owner"
                  values={{ name }}
               />
            </Fragment>
         )}
 </p>
```